client/disputes/details/index.tsx

```diff
@@ -41,7 +41,7 @@ const DisputeDetails = ( {
 			}
 			isSubmitted={
 				disputeObject.evidence_details &&
-				0 < disputeObject.evidence_details.submission_count
+				0 < ( disputeObject.evidence_details.submission_count ?? 0 )
```

Comment:
Looks like we could revert this (likely leftovers from types definitions experiments).